vdr/ambassador.go

```diff
@@ -320,6 +320,11 @@ func (n *ambassador) handleUpdateDIDDocument(transaction dag.Transaction, propos
 	// Stable order for metadata.SourceTransactions (derived from unordered maps): makes it easier to analyse and test.
 	sortHashes(sourceTransactions)
 
+	if len(sourceTransactions) == 1 && currentDIDMeta.Deactivated {
```

Comment:
Why the check for len == 1?

Reply:
See previous discussion. If length > 1 then there's a conflict which is "allowed"